Podverse/AudiosearchClientSwift.swift

```diff
@@ -67,5 +67,39 @@ class AudioSearchClientSwift {
             }
         }
     }
+    
+    static func retrievePodcast(id: Int64, onCompletion: @escaping ServiceResponseAny) -> Void {
+        let urlString = "shows/" + String(id)
```

Comment:
Not a big deal but just for swift correctness, instead of creating a whole new string object you can just create the urlString like this: [let urlString = "shows/\(id)"]